#126749 - add a filter to remove product and download from the saved … (#1245)
moderntribe/event-tickets: readme.txt

* Tweak - Update status manage to accept provider names or abbreviations [120862]
* Tweak - In the Ticket Block add link to EDD Orders Page [121440]
* Tweak - Change "Attendee Registration" to "Attendee Information" in several locations [126038]

> * Tweak - Exclude WooCommerce Product and EDD Downloads as supported post types for tickets to prevent recursion errors [126749]

Reviewer: ```suggestion
* Tweak - Exclude WooCommerce Product and EDD Downloads as supported post types when saving for tickets to prevent recursion errors, in case they were previously saved before we removed them from the options list [126749]
```

> * Tweak - Supported post types for tickets by removing product and download from saved options to prevent recursion errors [126749]

Reviewer: ```suggestion
* Tweak - Exclude WooCommerce Product and EDD Downloads as supported post types for tickets to prevent recursion errors [126749]
```

Reviewer: Is this fix only on save or does it no longer show them in the form either now?

Author: They were not showing in the form. They upgraded and the values were still saved from an older version.
